Validation/HGCalValidation/interface/HGVHistoProducerAlgo.h

```diff
@@ -155,7 +170,7 @@ class HGVHistoProducerAlgo {
   using Histograms = HGVHistoProducerAlgoHistograms;
 
   void bookInfo(DQMStore::IBooker& ibook, Histograms& histograms);
-  void bookCaloParticleHistos(DQMStore::IBooker& ibook, Histograms& histograms, int pdgid);
+  void bookCaloParticleHistos(DQMStore::IBooker& ibook, Histograms& histograms, int pdgid, unsigned layers);
```

Comment:
while `unsigned` is valid, maybe `unsigned int` would be preferable?

Reply:
Ok, changed accordingly everywhere.